src/main/java/uk/gov/justice/hmpps/prison/service/AdjustmentService.java

```diff
@@ -16,7 +16,7 @@ public class AdjustmentService {
     private final OffenderBookingRepository offenderBookingRepository;
 
     @Transactional(readOnly = true)
-    @VerifyBookingAccess
+    @VerifyBookingAccess(overrideRoles = {"VIEW_PRISONER_DATA"})
```

Comment:
hmpps-unused-deductions-listener uses this endpoint in dev (I missed this because it hadn't been put into prod yet)